api/controllers/UserController.js

```diff
@@ -253,41 +253,53 @@ module.exports = {
       console.log(err);
       return res.status(500).json(err);
     }
-    Reddit.getBothFlairs(req.user.redToken, req.params.username).then(function (flairs) {
-      var logged_fcs = user ? user.loggedFriendCodes : [];
+    var flairs;
+    flairs = await Reddit.getBothFlairs(req.user.redToken, req.params.username).catch(function (err) {
+      // Reddit will return 403 when looking up the flair of a user with a deleted account.
+      if (err.statusCode !== 403) {
+        throw err;
```

Comment:
Also, do we want to throw the error in the controller, probably not, we'll want to res.serverError probably